kit/src/components/user/mod.rs

```diff
@@ -41,6 +42,10 @@ pub fn get_time_ago(cx: &Scope<Props>) -> String {
     f.convert(duration)
 }
 
+pub fn get_aria_label(cx: &Scope<Props>) -> String {
```

Comment:
I don't like the pattern of creating a function to clone/unwrap/call something from `cx.props`, especially if it only gets used once. I think it's better to just call `cx.props.aria_label.clone().unwrap_or_default()` once to create the needed variable.